Turnstile-Ontology/99-Utils/Data/SoftwareUnitTestResult.csv

```diff
@@ -1,10 +1,10 @@
-"identifier_TEST_STATUS","identifier_SoftwareUnitTestExecution","identifier_SoftwareUnitTest","identifier_SoftwareUnitTestResult"
-"Failed","SoftwareUnitTestRun2","UTC-1-4","UTR-1-4-2"
-"Passed","SoftwareUnitTestRun2","UTC-1-3","UTR-1-3-2"
-"Passed","SoftwareUnitTestRun2","UTC-1-2","UTR-1-2-2"
-"Passed","SoftwareUnitTestRun2","UTC-1-1","UTR-1-1-2"
-"Failed","SoftwareUnitTestRun1","UTC-1-4","UTR-1-4-1"
-"Passed","SoftwareUnitTestRun1","UTC-1-3","UTR-1-3-1"
-"Passed","SoftwareUnitTestRun1","UTC-1-2","UTR-1-2-1"
-"Passed","SoftwareUnitTestRun1","UTC-1-1","UTR-1-1-1"
+"identifier_TEST_STATUS","generatedAtTime_SoftwareUnitTestResult","identifier_SoftwareUnitTestExecution","identifier_SoftwareUnitTest","identifier_SoftwareUnitTestResult"
+"Failed",null,"SoftwareUnitTestRun2","UTC-1-4","UTR-1-4-2"
+"Passed",null,"SoftwareUnitTestRun2","UTC-1-3","UTR-1-3-2"
+"Passed",null,"SoftwareUnitTestRun2","UTC-1-2","UTR-1-2-2"
+"Passed",null,"SoftwareUnitTestRun2","UTC-1-1","UTR-1-1-2"
+"Failed",null,"SoftwareUnitTestRun1","UTC-1-4","UTR-1-4-1"
+"Passed",null,"SoftwareUnitTestRun1","UTC-1-3","UTR-1-3-1"
+"Passed",null,"SoftwareUnitTestRun1","UTC-1-2","UTR-1-2-1"
+"Passed","Wed Mar 22 20:00:00 EDT 2017","SoftwareUnitTestRun1","UTC-1-1","UTR-1-1-1"
```

Comment:
This timestamp doesn't seem to be the same as the one in CounterApplicationUnitTesting.sadl, even allowing for timezone conversions?